### DISCUSSION TITLE
Ignoring folder locally

### DISCUSSION THREAD

**Discussion by nexpid** [community/community#62872] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello, in my github repo I have some files that are around 300mB. Is there a way to ignore them when I run `git pull`? (they're all in the assets directory)

**Comment by w8ste** [comment]
@Gabe616 As far as i know it is not possible to not pull or fetch the changes to a remote file (obviously ignoring is possible, but not what you want if i understand your question correctly). But doesn't git "ignore" files which are not changed when pulling ?

**Comment by nexpid** [comment]
It does, however I just don't want that many big files on my computer

**Reply by w8ste** [reply, marked as answer]
@Gabe616  Okay, well as i said i believe there aren't any options like this (would like the be corrected ^^). In my opinion your best option is to delete them every time you pull. I mean should not take to long if they are all in one directory :)

**Reply by nexpid** [reply]
that's true, thanks!

**Reply by w8ste** [reply]
You would have to run a `git restore assets` before resuming with your work so that deleting files would not cause problems with git
